ballerina/tests/basic-tests.bal

```diff
@@ -250,19 +248,17 @@ function testComplexTypes2() returns error? {
         quantity: 5
     };
     MedicalNeedClient mnClient = check new ();
-    int? id = check mnClient->create(need);
-    test:assertTrue(id is int);
+    MedicalNeed need2 = check mnClient->create(need);
+    test:assertTrue(need2.needId is int);
 
-    if id is int {
-        stream<MedicalNeed, error?> needStream = mnClient->read({itemId: 1});
-        int count = 0;
-        _ = check from MedicalNeed need2 in needStream
-            do {
-                test:assertEquals(need2.itemId, 1);
-                count = count + 1;
-            };
-        test:assertTrue(count > 0);
-    }
+    stream<MedicalNeed, error?> needStream = mnClient->read({itemId: 1});
```

Comment:
shall we change this to use query syntax

Reply:
Will be done via a separate PR